images.json

```diff
@@ -0,0 +1,37 @@
+[
+    {
+      "distro": "ubuntu",
+      "version_name": "noble",
+      "version_number": "24.4"
```

Comment:
I would include here the image's `url`, it is easier to change and update from the json file list

Reply:
So the way I did this is by having both an optional `url` property that lets you override it per image entry, and a `urls.json` that defines the templates based on `distro` so we dont have to copy paste the same URL every time.